tests/test_vision_service.py

```diff
@@ -16,12 +16,10 @@
 from viam.services.vision import (
     Detection,
     Classification,
-    VisModelConfig,
```

Comment:
All references to this dataclass are now gone. Is this dataclass still needed?

Reply:
can delete as well